brain/circuit.h

```diff
@@ -61,10 +61,8 @@ class Circuit : public boost::noncopyable
 
     /**
      * @return The set of GIDs for the given target name. If empty it will
```

Comment:
The brain::Circuit should not deal with targets. There is brion::Target class for that. Circuit::getGIDs() should return all GIDs of the circuit. Simple, easy, there you go!